harmony-fe/src/pages/_app.js

```diff
@@ -0,0 +1,5 @@
+import '../styles/global.css';
+
+export default function App({ Component, pageProps }) {
+  return <Component {...pageProps} />;
+}
```

Comment:
Could we convert this file into `.ts`? All we would need to change is the file extension and put a type on the args I think

i.e. `export default function App({ Component, pageProps }: WhateverThisTypeIs) {`

Reply:
Found it online, looks like we could probably just copy what is on this page: https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#with-typescript

Might need to be `.tsx` too? Which wouldn't mean any changes except for the file extension.